source/components/molecules/SwitchAction/SwitchAction.js

```diff
@@ -0,0 +1,96 @@
+/* eslint-disable no-nested-ternary */
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components/native';
+import { Button, Text } from 'source/components/atoms';
+import EditableList from '../EditableList';
+
+const InputWrapper = styled.View`
+  padding-bottom: 16px;
+`;
+
+const InpulList = props => (
+  <InputWrapper>
+    <EditableList {...props} />
+  </InputWrapper>
+);
+
+const SwitchAction = props => {
```

Comment:
Maybe the name of the component should just be FormField or Field? Hard to tell what the component is doing based on the name SwitchAction :D